library/Fission/CLI.hs

```diff
@@ -38,15 +38,11 @@ cli baseCfg = liftIO do
 
     runConnected_ :: Command FissionConnected input () -> Command.Leaf
     runConnected_ =
-      runWith \actn -> do
-        result <- runConnected baseCfg do
+      runWith \actn ->
+        void $ runConnected baseCfg do
```

Comment:
Oh, the error handling moved inside. Are we sure that we want to do the logging inside the action, and not out here?

Reply:
i.e. keep the error logging together

Reply:
(opposite of this)

Reply:
(also to be clear — not saying “not this”, just “have we considered both approaches?“)

Reply:
> Oh, the error handling moved inside
It actually already was inside. `put'` prints a generic "Something went wrong" error message (but it never actually gets printed cause it's run in `runConnected` which throws the error again)

I can see both arguments. My thought was that we already had all of the logs inline, so there seemed to be some desire for that style. Plus it gives a bit more flexibility with error handling. For instance `CLI.Error.notConnected` prints a diff message based on if the user has upgraded their CLI from an old pre-key version (granted, we could handle this with error types as well)

So yeah I can really go either way :man_shrugging:

Reply:
Yup, makes sense. It’s sometimes nice to propagate the error up to be able to handle or interpret it (for example, to know which command you’re in), but I can see both arguments. These commands are so short lived and concrete that I can see bth arguments. Okay, cool let’s merge this!